src/extensions/wcf/Microsoft.DotNet.UpgradeAssistant.Extensions.WCFUpdater/WCFUpdateStep.cs

```diff
@@ -108,14 +107,14 @@ public Task<UpgradeStepInitializeResult> Initialize(IProject project)
             FindPath(project);
             if (_path.Count != 3 && _path.Count != 4)
             {
-                return Task.FromResult(new UpgradeStepInitializeResult(UpgradeStepStatus.Skipped, "Cannot find required file(s) for WCF update. The project is not applicable for automated update and this step will be skipped.", BuildBreakRisk.Low));
+                return Task.FromResult(new UpgradeStepInitializeResult(UpgradeStepStatus.Complete, "Cannot find required file(s) for WCF update. The project is not applicable for automated update and this step is complete.", BuildBreakRisk.Low));
             }
 
             // if the project is not applicable, skip the update step
             if (!WCFUpdateChecker.IsWCFUpdateApplicable(_path, Logger))
             {
-                Logger.LogInformation("This project is not applicable for updating to CoreWCF since references to WCF was not found. This step will be skipped.");
-                return Task.FromResult(new UpgradeStepInitializeResult(UpgradeStepStatus.Skipped, "WCF Update step is not applicable to this project and this step will be skipped.", BuildBreakRisk.Low));
+                Logger.LogInformation("This project is not applicable for updating to CoreWCF since references to WCF was not found. There is no more work needs to be done and this step is complete.");
```

Comment:
```suggestion
                Logger.LogInformation("This project is not applicable for updating to CoreWCF since references to WCF were not found. No more work needs to be done and this step is complete.");
```